### ISSUE TITLE
Events do not trigger a m.redraw like they would in Mithril.

### ISSUE THREAD

**Issue by JoshGalvin** [StephanHoyer/mithril-query#4]
I'm getting around this by manually calling view(ctrl) after the event but ideally this would behave like Mithril.

**Comment by StephanHoyer** [comment]
interesting idea.

This would require to have an instance of the controller to trigger redraw.

Normally I do this

```javascript
$output = mq(view(ctrl));
$output.click('button');
$output = mq(view(ctrl));
$output.should.have('div');
```

If query has also a reference to the controller result one could write:

```javascript
$output = mq(view, ctrl); // or even $output = mq(myModule);
$output.click('button');
$output.should.have('div');
```

Think that would be awesome! Is this what you want?

```javascript
$output = mq(view, ctrl);



```

**Comment by JoshGalvin** [comment]
Yeah! And your first strategy is the one I'm currently using.

**Comment by StephanHoyer** [comment]
I try to accomplish this on weekend. But also a PR would be nice. I think that is a pretty easy task.

**Comment by StephanHoyer** [comment]
Do you want to give it a try ;)

https://github.com/StephanHoyer/mithril-query#auto-rendering

**Comment by StephanHoyer** [comment]
Just released 0.4.0 for this feature
